FLT/NumberField/Completion/Finite.lean

```diff
@@ -25,9 +25,11 @@ variable (v : HeightOneSpectrum (𝓞 K))
 open IsLocalRing
 
 instance NumberField.instFiniteResidueFieldAdicCompletionIntegers :
-    Finite (ResidueField (v.adicCompletionIntegers K)) := sorry -- Issue FLT#450
-  -- follows from IsDedekindDomain.HeightOneSpectrum.ResidueFieldEquivCompletionResidueField
-  -- and the above example.
+    Finite (ResidueField (v.adicCompletionIntegers K)) := by
+    apply (
+      IsDedekindDomain.HeightOneSpectrum.ResidueFieldEquivCompletionResidueField (v := v) (K  := K)
+    ).toEquiv.finite_iff.mp
+    apply Ideal.finiteQuotientOfFreeOfNeBot v.asIdeal v.ne_bot
```

Comment:
Just a minor formatting issue and golfing.

```suggestion
    Finite (ResidueField (v.adicCompletionIntegers K)) := by
  apply (HeightOneSpectrum.ResidueFieldEquivCompletionResidueField K v).toEquiv.finite_iff.mp
  exact Ideal.finiteQuotientOfFreeOfNeBot v.asIdeal v.ne_bot
```

This snippet has been tested locally so feel free to commit it directly if you like.